src/main/java/edu/mit/dos/object/ObjectService.java

```diff
@@ -26,7 +26,6 @@
     @RequestMapping(value = "/object", method = RequestMethod.POST)
     public String create(@RequestParam("handle") String handle,
                          @RequestParam("uris") List<String> fileUris,
-                         @RequestParam("title") String title,
```

Comment:
I'm assuming you didn't have luck with the title?

Reply:
No, I'm going to work on fully implementing it in my next PR once this is done